### ISSUE TITLE
createHTMLDocument() - argument not optional causes crashes in Edge

### ISSUE THREAD

**Issue by bthibault** [remarkablemark/html-dom-parser#3]

**Comment by bthibault** [comment]
https://developer.mozilla.org/en-US/docs/Web/API/DOMImplementation/createHTMLDocument

Argument is mandatory - this causes IE edge , 11, 10 to crash

**Comment by bthibault** [comment]
Dont forget to update html-react-parser :)

**Comment by remarkablemark** [comment]
@bthibault Thanks for letting me know of this bug.

I just released `0.1.1` with a fix:
```sh
$ npm install html-dom-parser@0.1.1
```

Let me know if you experience any other issues.

**Comment by remarkablemark** [comment]
@bthibault [html-react-parser](https://github.com/remarkablemark/html-react-parser) has been updated:
```sh
$ npm install html-react-parser@0.3.5
```
